subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1361,6 +1361,129 @@ static int cmd_wifi_ap_stations(const struct shell *sh, size_t argc,
 	return 0;
 }
 
+static int __wifi_ap_args_to_params(const struct shell *sh, size_t argc, char *argv[],
```

Comment:
No need to use `__` in the function name.

Reply:
Okay